src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -235,9 +239,11 @@ public EC2PrivateKey resolvePrivateKey() throws IOException {
         if (sshKeysCredentialsId != null) {
             SSHUserPrivateKey privateKeyCredential = getSshCredential(sshKeysCredentialsId, Jenkins.get());
             if (privateKeyCredential != null) {
+                LOGGER.fine("private key resolved from sshCredential");
                 return new EC2PrivateKey(privateKeyCredential.getPrivateKey());
             }
         }
+        LOGGER.info("unable to resolve private key");
```

Comment:
This is going to appear in system logs with no context and make no sense. Was something supposed to “resolve [a] private key”? What? Why was it not resolved? Is something broken?